src/services/fetchers/PolygonIOCryptoPriceFetcher.ts

```diff
@@ -1,8 +1,8 @@
 import {inject, injectable} from 'inversify';
 import {Logger} from 'winston';
 
-import PolygonIOCryptoPriceService from '../PolygonIOCryptoPriceService.js';
```

Comment:
I see a lot of unrelated changes like this, is your IDE changing some files automatically?

Reply:
No, I changed manually because I looked at those files. Every little improvement I can do, I try it to do.

Reply:
what is the improvement here? all I see is changing order of import